xsuite/src/cli/installRustCmd.ts

```diff
@@ -12,12 +12,17 @@ export const addInstallRustCmd = (cmd: Command) => {
     .description(
       `Install Rust with rustup: toolchain ${rustToolchain} & target ${rustTarget}.`,
     )
+    .option(
+      "--toolchain <TOOLCHAIN>",
+      `Rust toolchain version, defaults to ${rustToolchain}`,
+    )
     .action(action);
 };
 
-const action = () => {
+const action = ({ toolchain }: { toolchain?: string }) => {
+  const toolchain2Install = toolchain ? toolchain : rustToolchain;
```

Comment:
`toolchain2Install` is not correct English. 

Maybe renaming `rustToolchain` into `defaultRustToolchain` and then:

```
toolchain ??= defaultRustToolchain
```

?

Reply:
Done